docker/nvidia/README.md

```diff
@@ -0,0 +1,17 @@
+# Setting up your DTK development container to leverage NVIDIA GPUs on the host
+```bash
+[HOST]$ cd docker/nvidia
+[HOST]$ ./setup_nvidia_docker_compose.py
+[HOST]$ docker-compose build
+[HOST]$ docker-compose -p $USER up -d
+[HOST]$ docker exec -it <container_name> bash
+[CONTAINER]$ cd $TRILINOS_DIR/DataTransferKit
+[CONTAINER]$ # etc.
+```
+
+# Requirements
+* NVIDIA Docker plugin.  See [installation guidelines](https://github.com/NVIDIA/nvidia-docker/wiki/Installation) and consider downloading binary packages from the [release page](https://github.com/NVIDIA/nvidia-docker/releases).
+* PyYAML, a YAML parser and emitter for Python.  Consider using your system package manager (e.g. `apt-get install python-yaml` or `yum install python-yaml`) or pip (`pip install pyyaml`).
+
```

Comment:
In the requirements, I would add that they need to have compatible driver installed on the host. It's obvious for us but it might not be for everybody.